src/navigator_data_ingest/base/utils.py

```diff
@@ -76,3 +77,20 @@ def parser_input_already_exists(
         )
         return True
     return False
+
+
+def determine_content_type(response: Response, source_url: str) -> str:
+    """Use the response headers and file extension to determine content type
+
+    Args:
+        response (Response): the request object from the file download
+        source_url (str): The defined source url
+
+    Returns:
+        str: chosen content type
+    """
+
+    content_type_header = response.headers["Content-Type"].split(";")[0]
```

Comment:
Do you have access to the first 5 bytes to check the [file magic](https://en.wikipedia.org/wiki/List_of_file_signatures)? 
```
25 50 44 46 2D 	%PDF-
```

Reply:
Potentially, although I don't think I want to go down that route at this stage. I was aiming for as minimal an update as possible and its already taken a whole day 🙃